### PR TITLE
[ISV-3899] Create new operatorcert entry point to wait for github PR labels

### PR DESCRIPTION
**Pull request by jedinym**

### CODE REVIEW

File: operator-pipeline-images/operatorcert/entrypoints/github_wait_labels.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    Wait until ALL wait conditions on PR labels hold
+
+    Args:
+        repository: PyGithub repository object
+        pull_request_id: Github pull request id
+        timeout_s: Timeout for waiting on label changes in seconds
+        wait_conditions: List of conditions that must hold for a wait to end
+        poll_interval_s: Interval between finished requests to GitHub API
+    """
+    assert poll_interval_s < timeout_s
```
Comment (kosciCZ) [inline operator-pipeline-images/operatorcert/entrypoints/github_wait_labels.py:156]: Might be good to add a human readable message as a second argument to this assert, smth like "timeout needs to be bigger than the poll interval"

**Review by kosciCZ — APPROVED** [review]
One nitpick, otherwise LGTM :rocket:

**Review by haripate — APPROVED** [review]
LGTM

**Comment by mporrato** [comment]
@jedinym please add the new entrypoint to `project.scripts` in the `pyproject.yaml` file before merging.

**Pull request merged**